build.gradle.kts

```diff
@@ -4,14 +4,14 @@ plugins {
 	checkstyle
 	`maven-publish`
 	signing
-	id("com.diffplug.spotless") version "6.21.0"
+	id("com.diffplug.spotless") version "6.25.0"
 	id("at.zierler.yamlvalidator") version "1.5.0"
-	id("org.sonarqube") version "4.3.1.3277"
-	id("org.shipkit.shipkit-changelog") version "1.2.0"
-	id("org.shipkit.shipkit-github-release") version "1.2.0"
-	id("com.github.ben-manes.versions") version "0.48.0"
-	id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
-	id("org.gradlex.extra-java-module-info") version "1.4.2"
+	id("org.sonarqube") version "4.4.1.3373"
+	id("org.shipkit.shipkit-changelog") version "2.0.1"
```

Comment:
This is a major version change. Are you sure the project is compatible?

Reply:
Looking through the changes (not much) the ONLY relevant change for consumers is that they're building with Gradle 7 now: https://github.com/shipkit/shipkit-changelog/compare/v1.2.0...v2.0.1#diff-40640fe1078ece83d7ea8fb67daacd77923a86d13447baf9769660b3b46f2eceR3. I think they bumped major because maybe they think building with Gradle 7 makes minimum Gradle 7. Or it's a mono-release-train within the Shipkit org and something else was bumped higher.

Note: The Gradle task input fix was done in 1.2.3.

Reply:
Mockito also updated it without a hitch:
https://github.com/mockito/mockito/pull/3248
and they've released since then.